{
  this.testBean.exceptional(new RuntimeException());
  verify(mockCollaborator).noArgsOnRuntimeExceptionMatch();
}
